lib/src/modules/conditionals/case.dart

```diff
@@ -119,6 +127,9 @@ class Case extends Conditional {
   List<Conditional>? get defaultItem => _defaultItem;
   List<Conditional>? _defaultItem;
 
+  /// Optional label for the default case block.
+  final String? defaultLabel;
```

Comment:
idea: what if we accepted a `name` instead, and used it to construct the default label, something like `default_$name`?  not sure if that's better or worse.  thoughts?